dccjsonvalidation/schema_tools.py

```diff
@@ -443,3 +473,85 @@ def validation_errors(schema_errors, **kwargs):
             error_string += f"{prepend_string}{error.message}\n"
 
     return error_string
+
+
+def walk_schema(schema_obj, schema_output, first_call):
+    """
+    Function: walk_schema
+
+    Purpose: Recursively walk through the schema dictionary and construct
+             output to be written to a JSON schema file. The recursion is
+             necessary because a schema dictionary can have varying levels of
+             nested dictionaries and lists.
+
+             There are operations that only get performed the first time the
+             function is called, so the first_call parameter should always be
+             set to True when called in open code.
+
+    Arguments: - The Python dictionary representation of a JSON schema
+               - A character string for the output that gets appended to
```

Comment:
Can you clarify what you are expecting for the argument `schema_output`? Is the character string descriptive? Also, is it python style to list the arguments this way instead of being explicit about the argument name?

Reply:
- https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings
- https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

Reply:
I've edited the docstring to clarify the parameters.